api/src/main/resources/workbench-api.yaml

```diff
@@ -5858,6 +5858,28 @@ definitions:
           (MiB).
         type: number
         format: double
+      gce_egress_mib:
```

Comment:
So is an alert created if a combination of egress from these sources adds up to a value exceeding the threshold, or only if a single one triggers it? Are we moving toward a system where there could be different thresholds or windows for different sources?

Reply:
The alert threshold is unchanged - it is still targeting the total egress. We're not actively moving towards a different model, this change is simply to provide more information to the oncall who is receiving the alert.

We could decide to have different thresholds later if we wanted. Looking at a different threshold for dataproc worker egress in particular could be interesting at some point, since it doesn't encounter any of the client/server interaction egress which causes most of our false positives today.